R/distribution_formulas.R

```diff
@@ -0,0 +1,192 @@
+#' Set up a new distribution for a data type
+#'
+#' @param data_type A string specifying the type of data that the distribution
+#'   will be fit to. Options are listed in the function call, where the first
+#'   option listed, i.e., `"index"` is the default.
+#' @param module An identifier to a C++ fleet module that is linked to the data
+#'   of interest.
```

Comment:
I am wondering if "module" should be the first input argument for this function and the next one?

Reply:
Maybe match the order listed in the Roxygen documentation with the order of the input argument in the function.